.github/workflows/CI.yml

```
@@ -67,39 +72,38 @@ jobs:
       run: cmake -G Ninja -DCMAKE_BUILD_TYPE=${{ matrix.configuration }} ..
     - name: Build
       working-directory: build
-      run: ninja
+      run: ninja -k0
     - uses: actions/upload-artifact@v1
-      if: matrix.gcc-ver == '8'
+      if: matrix.gcc-ver == '10'
       with:
         name: openag-${{ runner.os }}-${{ matrix.configuration }}
         path: build/client.so
 
   build-linux-clang:
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-20.04 # TODO: Change this to ubuntu-latest once it becomes 20.04.
```

I think it's fine to just leave it as `ubuntu-20.04`. We're testing on a wide set of compilers rather than for "compiles on the latest Ubuntu".

```suggestion
    runs-on: ubuntu-20.04
```